projects/packages/jetpack-mu-wpcom/src/features/wpcom-options-general/options-general.ts

```diff
@@ -55,10 +55,28 @@ const _wpcomBuildSiteUrl = ( fragment: DocumentFragment ) => {
 		const homeUrlLabel = __( 'Site Address (URL)', 'jetpack-mu-wpcom' );
 		const homeUrlRow = document.createElement( 'tr' );
 		homeUrlRow.innerHTML = `
-					<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
-					<td><input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" /></td>
-			`;
+			<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
+			<td>
+				<input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" />
+				<button class="button wpcom-add-custom-address-button">
```

Comment:
It feels a bit off on mobile: 
<img width="362" alt="Screenshot 2025-03-03 at 12 21 56" src="https://github.com/user-attachments/assets/ffe034f9-57ea-476f-81ca-6a8b4b083bcd" />

Reply:
Thanks for checking, fixed.